docs(examples): add missing READMEs (#926)
algolia/autocomplete: examples/starter-algolia/README.md

> # Autocomplete starter targeting an Algolia index

Reviewer: ```suggestion
# Autocomplete starter with Algolia index
```

> This example helps you quickly boostrap a basic Autocomplete experience targeting a product index hosted on Algolia.

Reviewer: ```suggestion
This example helps you quickly boostrap a basic Autocomplete experience using Algolia.
```

> <p align="center"><img src="capture.jpg" alt="A capture of the Autocomplete starter targeting an Algolia index" /></p>

Reviewer: ```suggestion
<p align="center"><img src="capture.jpg" alt="A capture of the Autocomplete with Algolia" /></p>
```
